index.html

```diff
@@ -192,8 +193,30 @@ <h4>Don't Forget These!</h4>
               Island
             </label>
             <div class="button">
-              <button type="submit" class="btn btn-info">Let's Travel</button>
+              <button type="button" class="btn btn-info" data-toggle="modal" data-target="#exampleModal">Let's Travel</button>
             </div>
+
+            <!-- Modal -->
+            <div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+              <div class="modal-dialog" role="document">
+                <div class="modal-content">
+                  <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel">Confirmation</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                      <span aria-hidden="true">&times;</span>
+                    </button>
+                  </div>
+                  <div class="modal-body">
+                    Email has been sent!
+                  </div>
+                  <div class="modal-footer">
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <!-- Modal End -->
```

Comment:
Nice implementation on modal. Try adding it to the more info sections above as extra practice.